test/e2e_node/shortened_grace_period_test.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"bytes"
+	"context"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/dynamic"
+	watchtools "k8s.io/client-go/tools/watch"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+	"strings"
+	"time"
+)
+
+var _ = SIGDescribe("Shortened Grace Period", func() {
```

Comment:
Your test is not being run in the node-e2e job:

```
E2eNode Suite: [It] [sig-node] Shortened Grace Period When repeatedly deleting pods shorter grace period of a second command overrides the longer grace period of a first command

Reason: skipped
```

Not sure why, https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/113883/pull-kubernetes-node-e2e-containerd/1775072289461637120, but you'll want to make sure it runs before this can be merged.

Reply:
Maybe the container was gone the first time it was deleted? I'll try to change it.
I don't know why the test is not executing, do you have any suggestions?

Reply:
you have to add `[NodeConformance]` to the test